package.json

```diff
@@ -24,28 +24,31 @@
   },
   "homepage": "https://github.com/pinterest/esprint#readme",
   "dependencies": {
-    "dnode": "^1.2.2",
+    "dnode": "https://github.com/christianvuerings/dnode/#e08e620b18c9086d47fe68e08328b19465c62fb7",
```

Comment:
Had to fork `dnode` since it was not compatible with Node >= 10: https://github.com/substack/dnode/pull/194

Reply:
Hopefully this is just temporary.